core/src/main/scala/co.blocke.scalajack/json/TokenReader.scala

```diff
@@ -15,12 +15,23 @@ class TokenReader(
 
   override def peek: TokenType = tokenTypes(position + 1)
 
+  override def showError(): String = {
+    val charPos = tokenOffsets(position)
+    val startPosOffset = if (charPos - 50 < 0) charPos else 50
+    val startPos = charPos - startPosOffset
+    val endPos = if (charPos + 50 > source.length) source.length else charPos + 50
+    val buf = new StringBuffer()
+    buf.append(source.subSequence(startPos, endPos).toString + "\n")
+    buf.append("-" * startPosOffset + "^")
```

Comment:
Will this still work if the JSON had multiple lines? I believe these are raw offsets relative to the beginning of the input, not relative to the beginning of the offending line.